Assets/Scripts/Player/PlayerController.cs

```diff
@@ -215,6 +221,30 @@ public class PlayerController : MonoBehaviour {
 			else
 				UpdateCrouchColliders(crouchCollider, humanCollider, crouchGroundCollider, humanGroundCollider);
 		}
+		
+		if (isCrouched && Mathf.Abs(xInput) > 0.1f) {
+					crawlingSound.UnPause();
+		}
+		else {
+			crawlingSound.Pause();
+		}
+
+		if (isGrounded && !transformation.IsTransforming && 
+			transformation.State == TransformationState.Wolf && Mathf.Abs(xInput) > 0.1f) {
+			wolfRunningSound.UnPause();
+		}
+		else {
+			wolfRunningSound.Pause();
+		}
+
+		if (!isCrouched && !isCrouching && isGrounded && !transformation.IsTransforming &&
+			transformation.State == TransformationState.Human && Mathf.Abs(xInput) > 0.1f) {
+			humanRunningSound.UnPause();
+		}
+		else {
+			humanRunningSound.Pause();
+		}
+
```

Comment:
Could this be simplified to this?
```suggestion
if (Mathf.Abs(xInput) > 0.1f && isGrounded) {
			if (isCrouched) {
				crawlingSound.UnPause();
				humanRunningSound.Pause();
			}
			else if (transformation.State == TransformationState.Wolf) {
				wolfRunningSound.UnPause();
			}
			else if (transformation.State == TransformationState.Human && !isCrouching) {
				humanRunningSound.UnPause();
			}
		}
		else {
			crawlingSound.Pause();
			wolfRunningSound.Pause();
			humanRunningSound.Pause();
		}
```

Reply:
Nice, much more efficient! :D